src/app/shared/models/dot-wizard-step/dot-wizard-step.model.ts

```diff
@@ -0,0 +1,4 @@
+export interface DotWizardStep {
+    component: any;
```

Comment:
```suggestion
    component: T;
```

Reply:
agree but in the wizard and in most places T will be equal to `any`